src/api/app/views/webui2/webui/project/show.html.haml

```diff
@@ -76,7 +76,7 @@
     = render partial: 'delete_project_dialog', locals: { project: @project }
 
     - if show_package_actions?
-      = render partial: 'new_package_modal', locals: { project: @project }
+      = render partial: 'create_package_modal', locals: { project: @project }
```

Comment:
This looks wrong:wink:

Reply:
What is wrong? I changed the partial filename to better reflect its content.

Reply:
Oh, in the GitHub UI it looked like the partial name stayed the same. I overlooked `-> $new_file` part.

Sorry for the noise

Reply:
No worries